docs/guide.md

```diff
@@ -0,0 +1,210 @@
+# Getting Started with PatternPack
+
+## Creating a New PatternPack
+The idea behind PatternPack is to extract the styles from your application and move them into a separate repository where they can be shared between one or multiple projects.
+
+You have multiple methods to generate a Pattern Library:
+
+- Build your own configuration (covered in this section)
+- Start with the [example library](https://github.com/patternpack/patternpack-example-library) ([documentation](https://github.com/patternpack/patternpack/blob/master/docs/getting-started.md#start-with-the-example-pattern-library))
```

Comment:
I assume you're changing the order here to advocate for starting with the example library or generators first? I'll be honest that with the v1.0.2 release and the auto-generation of files built in, I think it's easy enough to build your own that I would advocate for starting there.

Thoughts?